pkg/mcs/resourcemanager/server/server.go

```diff
@@ -226,6 +226,14 @@ func (s *Server) Close() {
 		}
 	}
 
+	s.BaseServer.GetClientConns().Range(func(key, value any) bool {
```

Comment:
How about moving it to pkg/mcs/server/server.go?

Reply:
extracted